help.md

```diff
@@ -17,18 +17,18 @@ Look in the docs for an answer at [docs.pulpproject.org](https://docs.pulpprojec
 
 For documentation on a specific version of Pulp, see [Versioned Documentation](/docs/).
 
-## Issue Tracker
+## Bug or Feature Request?
 
-Look to see if the current bug or feature exists on the [Pulp issues tracker](https://pulp.plan.io/issues?set_filter=1).
+Think you've found a bug, [file it](https://pulp.plan.io/projects/pulp/issues/new).
 
-See bugs confirmed this week and what versions they affect with the [Weekly Bug Report](https://pulp.plan.io/projects/pulp/wiki/Weekly_Bug_Trends).
```

Comment:
This weekly report hasn't been updated since 2018 and I'm not sure how it benefits users looking for help anyway.

Reply:
AHAHAHA